packages/design/src/colors.css

```diff
@@ -58,33 +58,37 @@
   --color-overlay--dimmed: rgba(var(--color-white--rgb), 0.6);
 
   /* Brand */
-  --color-brand: var(--color-green);
+  --color-brand: hsl(79, 85%, 37%);
```

Comment:
Brand and Workflow colors are now being specified as raw values since they no longer correspond to underlying `base` values